microsoft/testsuites/core/hv_module.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy import assert_that
+
+from lisa import Logger, RemoteNode, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.operating_system import CentOs, Redhat
+from lisa.tools import Dmesg, Lsmod, Modinfo, Modprobe, Rpm, Uname
+from lisa.util import SkippedException
+
+
+@TestSuiteMetadata(
+    area="core",
+    category="functional",
+    description="""
+    This test suite covers test cases previously handled by LISAv2:
+    LIS-MODULES-CHECK, VERIFY-LIS-MODULES-VERSION,
+    INITRD-MODULES-CHECK, RELOAD-MODULES-SMP
+    """,
+)
+class HvModule(TestSuite):
+    def get_modules(self, node: RemoteNode) -> list[str]:
```

Comment:
Because it's not a test case, make it private, and put it after other test case methods.

Reply:
The method name can be improved, to specify get what kind of modules.